frontend/src/components/6.Swiping/SwipingPage.css

```diff
@@ -46,3 +46,9 @@
 #NoButton:hover {
   background-color: #d66354;
 }
+
+.ActionableSwipeCard {
+  z-index: 90;
```

Comment:
Right now, if you press the "Help" button on the Swiping Page, the card is rendered on top and the modal is unable to be closed until the delay between you swiping the card away and it reappearing with the voted tag.

This z-index value should either be changed to something lower, or you can give `.Help` a higher value e.g. 200 in `Common/Help.css`

Reply:
swt, changed z-index